perception/camera_obstacle_detection/yolo_v3/tensorflow_fp32_coco/execute_model/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2020, Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.0)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_STANDARD 14)
+
+project( example_pipeline )
```

Comment:
Can you change the name of the project here?

Reply:
Changed